doc/admin.md

```diff
@@ -248,15 +248,19 @@ the `mongob` section (as described bellow). E.g.:
 
 #### `mongodb`
 
-It configures the MongoDB driver for those repositories with 'mongodb' type. If the `host` parameter is a list of
-comma-separated IPs, they will be considered to be part of a Replica Set. In that case, the optional property
-`replicaSet` should contain the Replica Set name. If the database requires authentication, username (`user`), password
-(`password`) and authSource (`authSource`) can be set. If the database requires TLS/SSL connection but any validation of
-the certificate chain is not mandatory, all you need is to set the ssl (`ssl`) option as `true` to connect the database.
-If you need to add more complex option(s) such as `retryWrites=true` or `w=majority` when connection database, extraArgs
-(`extraArgs`) can be used to perform it. For The MongoBD driver will retry the connection at startup time `retries`
-times, waiting `retryTime` seconds between attempts, if those attributes are present (default values are 5 and 5
-respectively). E.g.:
+It configures the MongoDB driver for those repositories with 'mongodb' type.
+
+`uri` parameter overrides the Mongo options below, which have been deprecated. Mongodb URI format is available in
+`http://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html`.
+
+If the `host` parameter is a list of comma-separated IPs, they will be considered to be part of a Replica Set. In that
+case, the optional property `replicaSet` should contain the Replica Set name. If the database requires authentication,
+username (`user`), password (`password`) and authSource (`authSource`) can be set. If the database requires TLS/SSL
+connection but any validation of the certificate chain is not mandatory, all you need is to set the ssl (`ssl`) option
+as `true` to connect the database. If you need to add more complex option(s) such as `retryWrites=true` or `w=majority`
+when connection database, extraArgs (`extraArgs`) can be used to perform it. For The MongoBD driver will retry the
+connection at startup time `retries` times, waiting `retryTime` seconds between attempts, if those attributes are
+present (default values are 5 and 5 respectively). E.g.:
```

Comment:
```suggestion
It configures the MongoDB driver for those repositories with 'mongodb' type, using the `uri` parameter (which format is available in [this referece](http://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html)).
```

Me feeling is that deprecated configuration should not be mentioned. If some user needs documentation about it, it should check the documentation associated to an old version.

Note that's the way we use in other places (eg. Context Broker) https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/deprecated.md

> Documentation on deprecated features is removed from the repository documentation. Documentation is still available in the documentation set associated to older versions.